src/pages/api/identities/[id]/index.ts

```diff
@@ -68,8 +68,10 @@ const get: Handler = async (request, response) => {
       identityId,
     }).catch((error) => {
       span.setDisclosedAttribute("is Identity fetched", false);
-
-      if (error instanceof IdentityNotFoundError) {
+      if (
+        error instanceof IdentityNotFoundError ||
+        error.message === "Invalid UUID format"
+      ) {
```

Comment:
Atm, when the id in the url is a valid UUID but the identity doesn't exist, it throws a 404.
But when it is **not** a valid UUID it throws a 500. I think the fact that use user writes a valid UUID or not should not be treated differently than any other query string.